package.json

```diff
@@ -11,7 +11,8 @@
     "start-logging": "node app.js | ./node_modules/.bin/bunyan",
     "lint": "eslint --ext .js,.json .",
     "test": "NODE_ENV=test mocha --recursive test",
-    "watch-dev": "nodemon app.js | ./node_modules/.bin/bunyan",
+    "build-css": "./css-build",
+    "watch-dev": "nodemon app.js | ./node_modules/.bin/bunyan | ./css-watch",
```

Comment:
Originally I had this all on one line, but wasn't sure if this was best practice. I'll go with your suggestion.